frontend/components/modals/LockedModal.tsx

```diff
@@ -294,10 +296,14 @@ function LockedModalButton({
                 }
                 className="m-4"
               >
-                Unstake all
+                <span className="relative inline-flex items-center whitespace-nowrap">
```

Comment:
`<span className="relative inline-flex items-center whitespace-nowrap">` could be refactored I guess